docs/en/setup/backend/backend-receivers.md

```diff
@@ -148,6 +148,22 @@ receiver-meter:
   default:
 ```
 
+To activate the meter rule files:
+```yaml
+agent-analyzer:
+  selector: ${SW_AGENT_ANALYZER:default}
+  default:
+    meterAnalyzerActiveFiles: ${SW_METER_ANALYZER_ACTIVE_FILES:} # Which files could be meter analyzed, files split by ","
+```
+
+The receiver adds labels with `key = service` and `key = instance` to the collected data samples,
+and values from service and service instance name defined in SkyWalking Agent,
+for identification of the metric data.
+
+| Rule Name | Description | Configuration File | Data Source |
```

Comment:
I think this table belongs to `backend-meter.md`? As this config not belonging to the receiver.

Reply:
Changed to the `backend-meter.md`